UM/View/RenderBatch.py

```diff
@@ -234,43 +229,7 @@ def render(self, camera: Optional[Camera]):
 
         self._shader.release()
 
-    def _renderItem(self, item: Dict[str, Any]):
-        transformation = item["transformation"]
-        mesh = item["mesh"]
-
-        # Do not render if there's no vertex (empty mesh)
-        if mesh.getVertexCount() == 0:
-            return
-
-        normal_matrix = item["normal_transformation"]
-        if mesh.hasNormals() and normal_matrix is None:
-            normal_matrix = Matrix(transformation.getData())
-            normal_matrix.setRow(3, [0, 0, 0, 1])
-            normal_matrix.setColumn(3, [0, 0, 0, 1])
-            normal_matrix.invert()
-            normal_matrix.transpose()
-
-        self._shader.updateBindings(
-            model_matrix = transformation,
-            normal_matrix = normal_matrix
-        )
-
-        if item["uniforms"] is not None:
-            self._shader.updateBindings(**item["uniforms"])
-
-        vertex_buffer = OpenGL.getInstance().createVertexBuffer(mesh)
-        vertex_buffer.bind()
-
-        if self._render_range is None:
-            index_buffer = OpenGL.getInstance().createIndexBuffer(mesh)
-        else:
-            # glDrawRangeElements does not work as expected and did not get the indices field working..
-            # Now we're just uploading a clipped part of the array and the start index always becomes 0.
-            index_buffer = OpenGL.getInstance().createIndexBuffer(
-                mesh, force_recreate=True, index_start = self._render_range[0], index_stop = self._render_range[1])
-        if index_buffer is not None:
-            index_buffer.bind()
-
+    def _setMeshAttributes(self, mesh: Any):
```

Comment:
```suggestion
    def _setMeshAttributes(self, mesh: Any) -> None:
```
Partially typed functions are not checked by type checking.